docs/content/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Dagster Quickstart | Dagster Docs
+description: Run dagster for the first time
+---
+
+# Getting Started with Dagster
+
+<Note>
+  Looking to scaffold a new project? Check out the{" "}
+  <Link href="/getting-started/create-new-project">Creating a New Project</Link>{" "}
+  guide!
+</Note>
+
+## Introduction
+
+Welcome to Dagster! This guide will help you quickly run the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) project, showcasing Dagster's capabilities and serving as a foundation for exploring its features. The [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) project can be used without installing anything on your machine by using the pre-configured [GitHub Codespace](https://github.com/features/codespaces)! But if you prefer to run things on your own machine, we've got you covered.
+
+<TabGroup>
+<TabItem name="Option 1: Using GitHub Codespaces">
+
+### Option 1: Using GitHub Codespaces
+
+1. Fork the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) repository
+
+2. Select **Create codespace on main** from the **Code** dropdown menu.
+
+<Image
+width={400}
+height={400}
+alt="Create codespace"
+src="/images/getting-started/quickstart/github-codespace-create.png"
+/>
+
+3. After the codespace loads, start Dagster by running `dagster dev` in the terminal:
+
+   ```bash
+   dagster dev
+   ```
+
+4. Click **Open in Browser** when prompted.
+
+<Image
+width={400}
+height={300}
+alt="Codespace Open In Browser"
+src="/images/getting-started/quickstart/github-codespace-open-in-browser.png"
+/>
+
+5. **Success!**
+
+</TabItem>
+<TabItem name="Option 2: Running Locally">
+
+### Option 2: Running Locally
+
+<DagsterVersion />
+
+Ensure you have one of the supported Python versions installed before proceeding.
```

Comment:
Maybe worth linking to an installation page?

Reply:
Added a link, thanks.